css/style.css

```diff
@@ -121,10 +129,608 @@
+  position: static;
+  width: 375px;
+  height: 585px;
+  margin: 7px;
+  background-image: url("../assets/projectBackground1.png");
+  background-size: initial;
+  background-attachment: initial;
+  background-position: 39px 50px;
+  background-repeat: no-repeat;
+}
+
+.proArt {
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 39px;
+  line-height: 44px;
+  padding-left: 68px;
+  padding-top: 84px;
+  color: #fff;
+}
+
+.story2 {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  padding-left: 70px;
+  padding-top: 3px;
+  margin-top: -23px;
+}
+
+.button {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-left: 104px;
+  padding-right: 131px;
+  text-align: center;
+  text-decoration: none;
+  margin-top: 10px;
+  margin-left: 39px;
+  color: #fff;
+  background: gray;
+  cursor: pointer;
+}
+
+.grid-container2 {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 5px;
+  padding-left: 71px;
+  margin-top: -9px;
+}
+
+.stack2 {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  text-align: center;
+  background: rgba(255, 255, 255, 0.24);
+  line-height: 20px;
+  position: static;
+  color: #fff;
+  padding: 10px 12px;
+}
+
+.third {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 40px;
+  position: static;
+  width: 375px;
+  height: 568px;
+  margin: -201px;
+  background-image: url(../assets/projectBackground1.png);
+  background-size: initial;
+  background-attachment: initial;
+  background-position: 39px 50px;
+  background-repeat: no-repeat;
+}
+
+.fourth {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 40px;
+  position: static;
+  width: 375px;
+  height: 568px;
+  margin: 16px;
+  background-image: url(../assets/projectBackground1.png);
+  background-size: initial;
+  background-attachment: initial;
+  background-position: 39px 50px;
+  background-repeat: no-repeat;
+}
+
+.fifth {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 40px;
+  position: static;
+  width: 375px;
+  height: 568px;
+  margin: -206px;
+  background-image: url("../assets/projectBackground1.png");
+  background-size: initial;
+  background-attachment: initial;
+  background-position: 39px 50px;
+  background-repeat: no-repeat;
+}
+
+.sixth {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-top: 40px;
+  position: static;
+  width: 375px;
+  height: 568px;
+  margin: 16px;
+  background-image: url(../assets/projectBackground1.png);
+  background-size: initial;
+  background-attachment: initial;
+  background-position: 39px 50px;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Your `.second`, `.third`, `.fourth`, `.fifth` and `.sixth` class stylings are exactly the same. Please, use DRY principles and use one class instead. Then add that class in the different HTML elements.